src/bun_js.zig

```diff
@@ -469,6 +469,10 @@ pub const Run = struct {
     }
 };
 
+pub fn anyUnhandled() bool {
```

Comment:
`src/bun_js.zig` shouldn't be used outside of `bun run <file>` or `bun <file>`

There can be many other unhandled exception trackers.